core/primitives/src/action/mod.rs

```diff
@@ -216,6 +247,7 @@ pub enum Action {
     DeleteKey(Box<DeleteKeyAction>),
     DeleteAccount(DeleteAccountAction),
     Delegate(Box<delegate::SignedDelegateAction>),
+    DeployGlobalContract(DeployGlobalContractAction),
     #[cfg(feature = "protocol_feature_nonrefundable_transfer_nep491")]
```

Comment:
Sounds like `protocol_feature_nonrefundable_transfer_nep491` isn't stable yet and only potentially as part of nightly. Just wanted to make sure the ordering is not going to mess up BorshSerialize

Reply:
correct, since nonrefundable_transfer is not stable yet we want to add our feature before that, otherwise stabilising nonrefundable_transfer later will break borsch serialization